### DISCUSSION TITLE
Test API key? Update frequency?

### DISCUSSION THREAD

**Discussion by stucka** [GSA-TTS/FAC#2263] [Q&A] [answered]
Is there a timeline for when the API becomes fully public, and before then, is there a means with which the regular public can start building against the API such as API keys that work with the TEST site?

I see references to URLs such as https://api.data.gov/TEST/audit-clearinghouse/v0/dissemination/general&limit=10&offset=0&audit_year=eq.2022 but it tells me my duly issued data.gov API key isn't authorized.

If I hit fac.api.gov per the limited docs on the site I'm told there's not enough input, e.g. https://api.fac.gov/general?limit=5

And -- is there an idea of how frequently /general will be updated? Docs refer to some data sets being updated weekly, but aren't clear as to what.

**Comment by jadudm** [comment, marked as answer]
Hi @stucka  ,

You can visit

https://fac.gov/developers

for more information. `/general` is a live view into the system. Completed, validated, public audits should appear there immediately.

If there is documentation that is out-of-sync with the URL above, please drop a [bug report](https://github.com/GSA-TTS/FAC/issues/new?assignees=&labels=Type%3A+Bug%2CStatus%3A+Triage%2CNeeds%3A+Confirmation&projects=&template=bug-template.yaml&title=FAC+-+Bug%3A+%5BYOUR+TITLE%5D), and we will work to remove references to pre-release URLs.

**Reply by stucka** [reply]
Thank you! After all that I belatedly realized I was sending POSTs.
